src/Management/src/EndpointBase/Middleware/EndpointMiddleware.cs

```diff
@@ -69,24 +69,30 @@ public virtual string Serialize(TResult result)
 
             return string.Empty;
         }
+
+        internal JsonSerializerOptions GetSerializerOptions(JsonSerializerOptions serializerOptions)
+        {
+            serializerOptions ??= new JsonSerializerOptions() { PropertyNamingPolicy = JsonNamingPolicy.CamelCase };
+            serializerOptions.IgnoreNullValues = true;
+            if (serializerOptions.Converters?.Any(c => c is HealthConverter) != true)
```

Comment:
Do you need another check for the metrics converter instead of checking only for HealthConverter.

Reply:
We could add that, but they only way for it to happen is if a developer configured adding `HealthConverter` in their app config (see src/Management/test/EndpointCore.Test/CloudFoundry/EndpointMiddlewareTest.cs line 120 for an idea what that looks like)

Reply:
So If they add a HealthConverter, then they would lose the default MetricsResponseConverter ....

Reply:
That's true, but is there any conceivable reason somebody would do that?

Reply:
No but if they did, our code should be consistent, either dont check and always add, or check each one separately and add